### DISCUSSION TITLE
TRIGGER_STATE ERROR in QRTZ_TRIGGERS

### DISCUSSION THREAD

**Discussion by viveknuna** [quartznet/quartznet#2052] [Q&A]
I could see there has been a performance issue in my SQL server database. I can see some triggers in the ERROR state.

Can I change them from ERROR to WAITING? Is this recommended? or does it have any side effects?

`UPDATE QRTZ_TRIGGERS SET [TRIGGER_STATE] = 'WAITING' WHERE [TRIGGER_STATE] = 'ERROR'`

**Quartz.net version: 3.5.0
.Net Version: .Net 6**

**Comment by viveknuna** [comment]
@lahma Will you help in providing some fruitful information?

**Reply by lahma** [reply]
Yes. It depends. Like why are you getting the errors.

**Reply by viveknuna** [reply]
@lahma its failing due to timeout

**Reply by lahma** [reply]
Cool
